scripts/action-entrypoint-generator/action-entrypoint-generator

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+"""
+Generate the entrypoint.sh files for our GitHub actions.
+"""
+import argparse
+import sys
+from pathlib import Path
+
+
+TEMPLATE = """#!/usr/bin/env bash
+set -euo pipefail
+# AUTOGENERATED FILE, DO NOT EDIT!
+# This file has been generated by the `action-entrypoint-generator` script
+# defined in `scripts/action-entrypoint-generator`. To make changes to this
+# file, modify the script and rerun it.
+
+# GitHub overrides $HOME. Set it back to the home directory of our `app` user,
+# otherwise the call to `git config` fails.
+export HOME=/home/app
+
+# Mark the current directory as safe. If we don't do this, git commands fail
+# because the source in $PWD is owned by a different user than our `app` user.
+git config --global --add safe.directory "$PWD"
+
+@UPDATE_GGSHIELD@
+args=("$@")
+@COMMAND@
+"""
+
+# This script contains the code used by the unstable actions to update ggshield
+UPDATE_GGSHIELD_CODE = (Path(__file__).parent / "update-ggshield.sh").read_text()
+
+COMMANDS_FOR_DOMAIN = {
```

Comment:
Should we use the term `vertical` instead ? 🤔

Reply:
Sounds good, changed to it.